docs/guides/serverless-port-code.mdx

```diff
@@ -78,14 +78,13 @@ pass_manager = generate_preset_pass_manager(
 @distribute_task(target={"cpu": 1})
 def transpile_parallel(circuit, pass_manager):
     """Distributed transpilation for an abstract circuit into an ISA circuit for a given backend."""
-    
+
     isa_circuit = pass_manager.run(circuit)
 
     return isa_circuit
 
 try:
     # Get backend
-    # Follow instructions at https://docs.quantum.ibm.com/guides/setup-channel#iqp to authenticate with your API token.
     service = QiskitRuntimeService(channel="ibm_quantum")
```

Comment:
This is legacy-only, so it's safe to put the `channel`. But is it necessary if they already saved their login?

Reply:
In legacy, if you don't use channel= and you have both Cloud and IQP logins, and you didn't save IQP as default, it'll default to cloud.  That's why we put channel="ibm_quantum" all over.